build/builder.sh

```diff
@@ -8,6 +8,11 @@ function init() {
   docker build --tag="${image}" "$(dirname $0)"
 }
 
+function cockroach_dir() {
+  local dir=$(cd $(dirname $0); pwd)
```

Comment:
Per my other comment, I think this would be better as a global variable:

```
# Absolute path to the toplevel cockroach directory.
cockroach_toplevel="$(dirname $(cd $(dirname $0); pwd))"
```